ouroboros-network-protocols/testlib/Ouroboros/Network/Protocol/BlockFetch/Codecs.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+
+module Ouroboros.Network.Protocol.BlockFetch.Codecs where
```

Comment:
Let's include these newtypes in `Ouroboros.Network.Protocol.BlockFetch.Test` it's confusing to have both `**.BlockFetch.Codec` and `**.BlockFetch.Codecs`.

The same for other mini-protocols.